test/utils/shared.ts

```diff
@@ -22,3 +25,50 @@
 export function generateUniqueName(baseName = "", length = 4): string {
   return `${baseName}${crypto.randomBytes(length).toString("hex")}`;
 }
+
+export async function createDatabase(frame: Frame) {
+  const dbId = generateUniqueName("db");
+  const collectionId = generateUniqueName("col");
+  const sharedKey = `${generateUniqueName()}`;
+  // create new collection
+  await frame.waitFor('button[data-test="New Collection"]', { visible: true });
+  await frame.waitForSelector('div[class="splashScreen"] > div[class="title"]', { visible: true });
+  await frame.click('button[data-test="New Collection"]');
+
+  // check new database
+  await frame.waitFor('input[data-test="addCollection-createNewDatabase"]');
+  await frame.click('input[data-test="addCollection-createNewDatabase"]');
+
+  // check shared throughput
+  await frame.waitFor('input[data-test="addCollectionPane-databaseSharedThroughput"]');
+  await frame.click('input[data-test="addCollectionPane-databaseSharedThroughput"]');
+
+  // type database id
+  await frame.waitFor('input[data-test="addCollection-newDatabaseId"]');
+  const dbInput = await frame.$('input[data-test="addCollection-newDatabaseId"]');
+  await dbInput.press("Backspace");
+  await dbInput.type(dbId);
+
+  // type collection id
+  await frame.waitFor('input[data-test="addCollection-collectionId"]');
+  const input = await frame.$('input[data-test="addCollection-collectionId"]');
+  await input.press("Backspace");
+  await input.type(collectionId);
+
+  // type partition key value
+  await frame.waitFor('input[data-test="addCollection-partitionKeyValue"]');
+  const keyInput = await frame.$('input[data-test="addCollection-partitionKeyValue"]');
+  await keyInput.press("Backspace");
+  await keyInput.type(sharedKey);
+
+  // click submit
+  await frame.waitFor("#submitBtnAddCollection");
+  await frame.click("#submitBtnAddCollection");
+}
+
+export async function savePolicy(frame: Frame) {
```

Comment:
>savePolicy [](start = 22, length = 10)

Save? If the Save button always has this id

Reply:
rename as "onClickSaveButton"